Script.md

```diff
@@ -0,0 +1,27 @@
+## Introducing Lodgel – the property management application that streamlines the rental process.
+Lodgel removes the hassle of ensuring your properties are ready for guests by coordinating assistants

```

Comment:
Please make this consistent with the language we use elsewhere -> turnover assistant

Reply:
✔️